yaml/com/ibm/License/Entry/LicenseEntry.interface.yaml

```diff
@@ -0,0 +1,67 @@
+description: >
+    Implement to represent a license which enables features on system
+    Customer installs license to unlock certain resources or features
+    Each object implements xyz.openbmc_project.License.Entry interface
+    which has Name and Status properties
+    and implements xyz.openbmc_project.Object.Delete interface.
+properties:
+
+    - name: Name
+      type: string
+      description: >
+         Name of license
+    - name: SerialNum
+      type: string
+      description: >
+         The serial number of the license
+    - name: Type
+      type: enum[self.Type]
+      default: Trial
+      description: >
+          The current type of the license
+    - name: AuthorizationType
+      type: enum[self.AuthorizationType]
+      default: Unlimited
+      description: >
+          The current Authorization type of the license
+    - name: ExpirationTime
+      type: string
```

Comment:
I don't think a string makes sense..
Maybe something like

    - name: ScheduledTime
      type: uint64
      default: 0
      description: >
          ScheduledTime is a date time when the host will be
          powered on/off. The time is the Epoch time, number
          of seconds since 1 Jan 1970 00::00::00 UTC.


https://github.com/openbmc/phosphor-dbus-interfaces/blob/08baf48ad5f15774d393fbbf4e9479a0ef3e82d0/yaml/xyz/openbmc_project/State/ScheduledHostTransition.interface.yaml

Reply:
but we have to take this string and convert it to some other format for Redfish. 
I really don't think a string makes sense here.

Reply:
@gtmills Seems like phyp will be sending the Date in YYYY-MM-DD format. We can go ahead with this string attribute now until that is going to change from Phyp. From the redfish perspective, this is still a string. So there should not be any validator error.

Reply:
Done. Changed to uint64